ITC205_BaseLine_Code/src/ReturnBookControl.java

```diff
@@ -19,7 +19,7 @@ public void setUI(ReturnBookUI ui) {
 			throw new RuntimeException("ReturnBookControl: cannot call setUI except in INITIALISED state");
 		}	
 		this.ui = ui;
-		ui.setState(ReturnBookUI.UI_STATE.READY);
+		ui.SetState(ReturnBookUI.UiState.READY);
 		state = CONTROL_STATE.READY;		
 	}
 

```

Comment:
D1 - delete blank line?

Reply:
Will amend ReturnBookControl in alternate branch. This is the next branch to be amended.